packages/ia-topnav/src/nav-search.js

```diff
@@ -0,0 +1,56 @@
+import { html } from 'lit-element';
+import TrackedElement from './tracked-element';
+import navSearchCSS from './styles/nav-search';
+import icons from './assets/img/icons';
+
+class NavSearch extends TrackedElement {
+  static get styles() {
+    return navSearchCSS;
+  }
+
+  static get properties() {
+    return {
+      config: { type: Object },
+      open: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.config = {};
+    this.open = false;
+  }
+
+  search(e) {
+    this.trackSubmit(e);
+    this.dispatchEvent(new CustomEvent('navSearch', {
+      detail: {
+        originalEvent: e,
+        formEl: this.shadowRoot.querySelector('form'),
```

Comment:
How come you're passing the whole form element up?

Reply:
Whole thing's been refactored now.